src/expressions.js

```diff
@@ -183,11 +183,11 @@ var _ = Mavo.Expressions = $.Class({
 	},
 
 	extract: function(node, attribute, path, syntax = Mavo.Expression.Syntax.default) {
-		if (attribute && _.skip.indexOf(attribute.name) > -1) {
+		if (_.skip.includes(attribute?.name)) {
```

Comment:
Can we remove this entirely now that `_.skip` is taken into account in `traverse()`? As far as our own codebase is concerned, that seems ok, but not sure if any plugins call `extract()` and depend on this behavior

Reply:
Good question. I searched for `extract(` in our repo with plugins and found no mentions. Neither could I find this in my repos. I'm unsure if we have any other plugins we might break. We can try to commit to this change and see what might happen. 🤓

Reply:
I'm ready to remove it as soon as we decide to do that. I commented this block out and everything still works.

Reply:
Let's do it!

Reply:
Done!

Now, I'm ready to request your review one more time. Thanks a ton for your patience and excellent ideas! 🙏🏻